src/wovnio/wovnphp/Utils.php

```diff
@@ -97,11 +97,11 @@ public static function isFilePathURI($uri, $store)
      * Return true if $uri should be ignored according to `ignore_paths` or `ignore_regex`.
      * Return false otherwise.
      */
-    public static function isIgnoredPath($uri, $store)
```

Comment:
How about always use URL with query.

Reply:
Hmm I guess that's OK. Initially I didn't want to do it because I worry that ignore_paths might not behave correctly. But after looking at the code i think it's ok.